packages/vue/src/components/authenticator.vue

```diff
@@ -224,41 +208,19 @@ const onConfirmVerifyUserSubmitI = (e: Event) => {
   }
 };
 
-// watchers
-
-/**
- * Update service facade when context updates
- */
-
-const user: Ref<AmplifyUser | null> = ref(null);
-const signOut = ref();
-
-watch(
-  () => state.value.context,
-  () => {
-    const { user: u, signOut: s } = getServiceFacade({
-      send,
-      state: state.value,
-    });
-    user.value = u;
-    signOut.value = s;
-  }
-);
-
 const hasTabs = computed(() => {
-  return (
-    actorState.value?.matches('signIn') || actorState.value?.matches('signUp')
-  );
+  return route.value === 'signIn' || route.value === 'signUp';
 });
 
 const hasRouteComponent = computed(() => {
-  return !(
-    state.value.matches('authenticated') ||
-    state.value.matches('idle') ||
-    state.value.matches('setup') ||
-    state.value.matches('signOut') ||
-    actorState.value?.matches('autoSignIn')
-  );
+  const routesWithoutComponents: AuthenticatorRoute[] = [
```

Comment:
Nittish: would prefer evaluation against `routesWithComponents` (and removing the bang on line 223). Idea being that if a new non-component route were added `hasRouteComponent ` would still evaluate as expected for the routes that render

Reply:
Agreed, done in 9fec28ccd842ead311f17a2a9de0fe6ea78b9e1f (and sorted in another commit)